src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react"
+import { Flex, Text, ModalDialog, Spacer, Button } from "@artsy/palette"
+import { SortFilter } from "Components/SortFilter"
+import { ArtworksList, TEMP_ARTWORKS } from "./ArtworksList"
+import { useTranslation } from "react-i18next"
+
+interface AddArtworksModalProps {
+  visible: boolean
+  onClose: () => void
+  onComplete: () => void
+  listName: string
+}
+
+const AddArtworksModal: React.FC<AddArtworksModalProps> = ({
+  listName,
+  onClose,
+}) => {
+  const { t } = useTranslation()
+
+  const handleSave = () => {
+    onClose()
+  }
+
+  return (
+    <ModalDialog
```

Comment:
header and footer were supposed to be sticky, but for some reason they scroll together with the content

https://user-images.githubusercontent.com/3513494/223394860-a985b85c-ed5a-4543-a07f-0de2fd76ea4d.mp4